### PR TITLE
Enabled OpenAPI for coverage with checkpoint and restore

### PR DESCRIPTION
**Pull request by KeenanMcCarthy**

### CODE REVIEW

**Comment by KeenanMcCarthy** [comment]
#build
#libby

File: dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/OpenAPITest.java [hunk 1]
```diff
@@ -52,22 +52,18 @@ public static void exportWebApp() throws Exception {
...
 
         ShrinkHelper.exportAppToServer(server, webappWar, DeployOptions.OVERWRITE);
+
+        ServerConfiguration config = server.getServerConfiguration();
+        config.getVariables().getById("openAPIserver").setValue("alternateServer");
+        server.updateServerConfiguration(config);
     }
 
     @Test
     @Ignore
```
Comment (tjwatson) [inline dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/OpenAPITest.java]: Need to remove the `@Ignore` annotation here.

**Review by tjwatson — APPROVED** [review]

**Comment by tjwatson** [comment]
#libby

**Comment by KeenanMcCarthy** [comment]
#build

**Pull request merged**